### ISSUE TITLE
http-signature Error: Cannot find module 'assert-plus'

### ISSUE THREAD

**Issue by Bagusa4** [microsoft/vscode-generator-code#91]
#Hellow, when I running `yo code` then I got this error, any suggestions?

```
module.js:529
    throw err;
    ^

Error: Cannot find module 'assert-plus'
    at Function.Module._resolveFilename (module.js:527:15)
    at Function.Module._load (module.js:476:23)
    at Module.require (module.js:568:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\%NAME%\AppData\Roaming\npm\node_modules\generator-code\node_modules\http-signature\lib\parser.js:3:14)
    at Module._compile (module.js:624:30)
    at Object.Module._extensions..js (module.js:635:10)
    at Module.load (module.js:545:32)
    at tryModuleLoad (module.js:508:12)
    at Function.Module._load (module.js:500:3)
```

thanks, before !

**Comment by zsichinava** [comment]
Same here

**Comment by StefanJanssen95** [comment]
I've got the same issue and fixed it locally as well, since @aeschli has been assigned since October on this issue and its a 2 minute fix I will make a pull request in a second to fix this.

**Comment by StefanJanssen95** [comment]
Solved by #103
